clients/js/src/fetchMetadataContent.ts

```diff
@@ -42,3 +76,34 @@ export async function fetchAndParseMetadataContent(
       return content;
   }
 }
+
+type FetchAndParseAllMetadataContentInput = FetchAllMetadataContentInput;
+
+export async function fetchAndParseAllMetadataContent(
+  rpc: Rpc<GetMultipleAccountsApi>,
+  input: FetchAndParseAllMetadataContentInput
+): Promise<unknown[]> {
+  const addresses = await Promise.all(
+    input.map(async ({ program, authority, seed }) => {
+      const [address] = await findMetadataPda({ program, authority, seed });
+
+      return address;
+    })
+  );
+  const maybeAccounts = await fetchAllMaybeMetadata(rpc, addresses);
+  const accounts = maybeAccounts.filter((acc) => acc.exists);
+  const unpacked = await unpackAndFetchAllData({ rpc, accounts });
+  return unpacked.map((content, index) => {
+    switch (accounts[index].data.format) {
+      case Format.Json:
+        return JSON.parse(content);
+      case Format.Yaml:
+        return parseYaml(content);
+      case Format.Toml:
+        return parseToml(content);
+      case Format.None:
+      default:
+        return content;
+    }
```

Comment:
Should we extract this logic in a **non-exported** helper function named something like `parseContent`?